ci-operator/step-registry/openshift/e2e/openstack/perfscale/conf-securitygroup/deprovision/openshift-e2e-openstack-perfscale-conf-securitygroup-deprovision-commands.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# For disconnected or otherwise unreachable environments, we want to
+# have steps use an HTTP(S) proxy to reach the API server. This proxy
+# configuration file should export HTTP_PROXY, HTTPS_PROXY, and NO_PROXY
+# environment variables, as well as their lowercase equivalents (note
+# that libcurl doesn't recognize the uppercase variables).
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+	# shellcheck disable=SC1090
+	source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
```

Comment:
Since `clouds.yaml` is not in a standard path, to avoid getting `Cloud openstack was not found` you have to set this variable:

```bash
export OS_CLIENT_CONFIG_FILE="${SHARED_DIR}/clouds.yaml"
```